lostmediafinder/finder.py

```diff
@@ -733,3 +733,36 @@ async def _run(cls, id, session: aiohttp.ClientSession):
                    name=cls.getName(), note=cls.note,
                    rawraw=None, metaonly=False, classname=cls.__name__
         )
+
+class LetsPlayIndex(YouTubeService):
+    name = methods['letsplayindex']['title']
+    note = ""
+    configId = "letsplayindex"
+
+    @classmethod
+    async def _run(cls, id, session: aiohttp.ClientSession):
+        url = f"https://www.letsplayindex.com/video/x-{id}"
+        archived = False
+
+        try:
+            async with session.head(url, timeout=15) as resp:
+                lastupdated = time.time()
+                status = resp.status
+                if status == 301:
+                    archived = True
+                    available = str(resp.request_info.url)
+                    yield Link(
+                        url = available,
+                        contains = LinkContains(video = True, metadata = True, thumbnail = True),
+                        title = "Video"
+                    )
+                elif status == 404:
+                    archived = False
+        except (aiohttp.ClientError, asyncio.TimeoutError):
+            archived = False
+            lastupdated = time.time()
```

Comment:
Do these exceptions occur when the video isn't archived, or is this just error handling? I'd rather not hide errors like this if they occur. Exceptions are fine to raise if necessary, they'll be shown in the UI as errors.